junifer/testing/datagrabbers.py

```diff
@@ -55,7 +55,10 @@ def get_item(self, subject: str) -> Dict[str, Dict]:
         """
         out = {}
         i_sub = int(subject.split("-")[1]) - 1
-        out["VBM_GM"] = {"path": Path(self._dataset.gray_matter_maps[i_sub])}
+        out["VBM_GM"] = {
+            "path": Path(self._dataset.gray_matter_maps[i_sub]),
+            "space": "MNI",
```

Comment:
I'd rather use the right space here, so we test correctly the masks/parecellations.